firebase.go

```diff
@@ -146,11 +151,7 @@ func (fb *Firebase) copy() *Firebase {
 }
 
 func (fb *Firebase) makeRequest(method string, body []byte) (*http.Request, error) {
-	path := fb.url + "/.json"
-
-	if len(fb.params) > 0 {
-		path += "?" + fb.params.Encode()
-	}
+	path := fb.String()
```

Comment:
I would say that we can just get rid of the function altogether. The purpose of this function was to append all of the query string parameters to the url but if we're already doing that in the `String()` method we don't need this method.